Kickstarter-iOS/Features/PledgePaymentMethods/Datasource/PledgePaymentMethodsDataSourceTests.swift

```diff
@@ -42,18 +42,18 @@ final class PledgePaymentMethodsDataSourceTests: XCTestCase {
   func testLoad_PaymentSheetCardValues() {
     let paymentSheetData = [
       (
-        image: UIImage(),
```

Comment:
Data source now uses the same data type for new cards and existing cards, as mentioned.